docs/src/DevelopersGuide.md

```diff
@@ -0,0 +1,142 @@
+# Developer's Guide
+
+This doc is meant to guide any new developers with organization/formatting and common bugs associated with CloudMicrophysics.jl.
+
+## GitHub Pull Requests (PR)
+
+After creating a new branch, create a PR on GitHub.
```

Comment:
```suggestion
We work on branches of the main repository, rather than personal forks.
Each branch name should ideally start with your initials, followed by short
  name relevant to what will be added in that branch. 
It is considered a good practice to open an issue that describes the planned work
  before starting the implementation.
This allows you to get feedback from other developers and advertise the planned work to the group.
When the implemented code is ready for review, create a pull request (PR) on GitHub
  and tag the issue it is addressing.
```